SPECS/linux-firmware.spec

```diff
@@ -75,6 +75,18 @@ cp %{SOURCE1} %{SOURCE2} amd-ucode/
 /lib/firmware/*
 
 %changelog
+* Tue Aug 08 2023 Gael Duperrey <gduperrey@vates.fr> - 20190314-9
```

Comment:
Small mistake missed in review: the release number should be 20190314-9.1.

Fix PR incoming.